docs/CEAP_en_US.md

```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
+1. BRL 1353,04, the Congress Person who's:
+  * Leader of a political party, parliament block, Minority, or leader of the Government in the Chamber of Deputies or the National Congress.
+  * Vice-leader of a political party or parliament block.
+  * President or vice-president of a Permanent Comission.
+  * Representative of political parties **with less members than a hundredth** of the total number of Congress People in the Chamber of Deputies.
+2. BRL 902,02, the Congress Person who's:
+  * Vice-leader of the Minority
+  * Vice-leader of the Government in the Chamber of Deputies or the National Congress.
+* If the same Congress Person occupies more than one of the above positions simultaneously, the extra quota **cannot be multiplied**.
+* The Congress Person that represents Brazil in a Mercosul parliamentary mission can spend an extra 20% of the smallest quota, for each trip.
+
+Article 2 - The quota can be used to cover the following expenses:
+* Airline tickes
+* Telephony services
+* Postal services and products, but **not the purchase of postal stamps and shipping of postal cards**
```

Comment:
What about _**excluding** the purchase of postal stamps and shipping of postal cards_?